packages/components/src/Status/Status.component.js

```diff
@@ -105,5 +97,3 @@ Status.propTypes = {
 Status.defaultProps = {
 	actions: [],
 };
-
-export default Status;
```

Comment:
Because of this, `Status` component is no more required like that : 

`import Status from 'blah';`

but like that : 

`import { Status } from 'blah';`

So I think this is a breaking change too. What do you think ?

Reply:
done